chart/values.yaml

```diff
@@ -469,3 +471,56 @@ rttys:
   tolerations: []
 
   affinity: {}
+
+# Kafka
+kafka:
+  enabled: true
```

Comment:
Why is the Kafka deployment enabled by default but the actual usage by uCentral disabled? IMO those two values should align in the default configuration.